gallery/lib/home.dart

```diff
@@ -0,0 +1,121 @@
+import 'package:flutter/material.dart';
+
+import 'demo.dart';
+import 'settings.dart';
+import 'splash.dart';
+
+class HomePage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text('Gallery'),
+        leading: IconButton(icon: Icon(Icons.color_lens), onPressed: () {
+          Navigator.of(context).push(
+            MaterialPageRoute(
+              fullscreenDialog: true,
+              builder: (context) => SplashPage(),
+            ),
+          );
+        }),
+        actions: <Widget>[
+          IconButton(icon: Icon(Icons.settings), onPressed: () {
+            Navigator.of(context).push(
+              MaterialPageRoute(
+                fullscreenDialog: true,
+                builder: (context) => SettingsPage(),
+              ),
+            );
+          }),
+        ],
+      ),
+      body: Padding(
+        padding: const EdgeInsets.all(16.0),
+        child: ListView(
+          children: <Widget>[
+            Text('Gallery', style: Theme.of(context).textTheme.headline,),
+            Container(
+              height: 200,
+              child: ListView(
+                scrollDirection: Axis.horizontal,
+                children: <Widget>[
+                  CarouselCard(title: 'Shrine'),
+                  CarouselCard(title: 'Rally'),
+                  CarouselCard(title: 'Crane'),
+                ],
+              ),
+            ),
+            Text('Categories', style: Theme.of(context).textTheme.headline,),
+            CategoryListItem(title: 'Material'),
+            CategoryListItem(title: 'Cupertino'),
+            CategoryListItem(title: 'Reference styles & Media'),
+          ],
+        ),
+      ),
+    );
+  }
+}
+
+class CarouselCard extends StatelessWidget {
+  const CarouselCard({Key key, this.title}) : super(key: key);
+
+  final String title;
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsetsDirectional.only(top: 8.0, end: 16.0, bottom: 8.0),
```

Comment:
You should be consistent about whether or not you include the trailing `.0` on double params like this. My vote is that you should remove it, but either way you should probably be consistent.

Reply:
Keeping the `.0` on all, I think that will be the style guide for this project.